_install-and-configure/configuring-opensearch/index-settings.md

```diff
@@ -257,6 +257,10 @@ OpenSearch supports the following dynamic index-level index settings:
 
 - `index.optimize_doc_id_lookup.fuzzy_set.false_positive_probability` (Double): Sets the false-positive probability for the underlying `fuzzy_set` (that is, the Bloom filter). A lower false-positive probability ensures higher throughput for upsert and get operations but results in increased storage and memory use. Allowed values range between `0.01` and `0.50`. Default is `0.20`.
 
+- `index.routing.allocation.total_shards_per_node` (Integer): Maximum number of primary and replica shards (total) from a single index that can be allocated on a single node. Defaults to `-1` (unlimited). Helps control per-index shard distribution across nodes by limiting shards per node. Use with caution as shards from this index may remain unallocated if nodes reach their configured limits.
```

Comment:
```suggestion
- `index.routing.allocation.total_shards_per_node` (Integer): The maximum combined total number of primary and replica shards from a single index that can be allocated to a single node. Default is `-1` (unlimited). Helps control per-index shard distribution across nodes by limiting the number of shards per node. Use with caution because shards from this index may remain unallocated if nodes reach their configured limits.
```